<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwigt</class>
 <widget class="QWidget" name="mainwigt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainwigt</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="2" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="gB_param">
     <property name="title">
      <string>参数区</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="pB_readparam">
        <property name="text">
         <string>读取参数</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pB_setparam">
        <property name="text">
         <string>设置参数</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTableView" name="tbV_param"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gB_cmd">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>命令区</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>↓校准按钮</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>    ↓校准值</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pB_J_1_1">
        <property name="statusTip">
         <string>0</string>
        </property>
        <property name="text">
         <string>电压1校准点1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_J_1_1"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pB_J_1_2">
        <property name="statusTip">
         <string>1</string>
        </property>
        <property name="text">
         <string>电压1校准点2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_J_1_2"/>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pB_J_1_3">
        <property name="statusTip">
         <string>2</string>
        </property>
        <property name="text">
         <string>电压1校准点3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="le_J_1_3"/>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pB_J_2_1">
        <property name="statusTip">
         <string>3</string>
        </property>
        <property name="text">
         <string>电压2校准点1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="le_J_2_1"/>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pB_J_2_2">
        <property name="statusTip">
         <string>4</string>
        </property>
        <property name="text">
         <string>电压2校准点2</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="le_J_2_2"/>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="pB_J_2_3">
        <property name="statusTip">
         <string>5</string>
        </property>
        <property name="text">
         <string>电压2校准点3</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="le_J_2_3"/>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="pB_J_I_1">
        <property name="statusTip">
         <string>6</string>
        </property>
        <property name="text">
         <string>电流校准点1</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="le_J_I_1"/>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="pB_J_I_2">
        <property name="statusTip">
         <string>7</string>
        </property>
        <property name="text">
         <string>电流校准点2</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="le_J_I_2"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="gB_sport">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>190</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>串口通信：</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="hlo_1">
        <item>
         <widget class="QLabel" name="lb_RTU">
          <property name="text">
           <string>串口：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cB_RTU">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlo_baud">
        <item>
         <widget class="QLabel" name="lb_baud">
          <property name="text">
           <string>波特率：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cB_baud">
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pB_findRTU">
        <property name="text">
         <string>搜寻串口</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlo_2">
        <item>
         <widget class="QLabel" name="lb_stateRTU">
          <property name="text">
           <string>未连接</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pB_linkRTU">
          <property name="text">
           <string>连接</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2" colspan="3">
    <widget class="QTextEdit" name="tB_log"/>
   </item>
   <item row="3" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>93</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pB_clearlog">
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="gB_date">
     <property name="title">
      <string>数据区</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>设备ID：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_ID">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pB_findID">
        <property name="text">
         <string>查找ID</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QTableView" name="tbV_value"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pB_readvalue">
        <property name="text">
         <string>读取实时数据</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chB_isloop">
        <property name="text">
         <string>周期读取/ms</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_looptime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>1000</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
